### ISSUE TITLE
Look controls hmdEnabled:false seems to be ignored

### ISSUE THREAD

**Issue by andreyrd** [aframevr/aframe#4520]
**Description:**
I would expect the configuration `enabled: true; hmdEnabled: false; touchEnabled: true;` to only take touch input, but gyroscope controls are still enabled. In issue #4095, the look-controls component is *entirely* disabled. But I would like touch controls to stay enabled while having the ability to toggle on & off gyroscope controls.

The source code for look-controls will reset yaw & pitch when you set `hmdEnabled` to `false`, but it will not actually do anything else with that setting, and will still read continue to read and apply data from hmd:
https://github.com/aframevr/aframe/blob/master/src/components/look-controls.js#L79

In past versions, I would rewrite look-controls and implement `hmdEnabled` myself. Last version I did this for was 0.9.2. If I'm understanding the purpose of `hmdEnabled` correctly, and this is a bug, I can open a PR for the latest version.

- A-Frame Version: 1.0.4
- Platform / Device: iOS 13.3.1 / iPhone 8, Android Chrome 80.0.3987.132 / Android 9 / motorola one zoom
- Reproducible Code Snippet or URL: https://rust-low-camp.glitch.me

**Comment by dmarcos** [comment]
yeah thanks. A PR will be appreciated. `hmdEnabled` might be also misleading. I would probably rename to `magicWindowTrackingEnabled`
